packages/runtime/test/transport/publicRelationshipTemplateReferences.test.ts

```diff
@@ -0,0 +1,50 @@
+import { ClientResult, PublicRelationshipTemplateReferenceClient } from "@nmshd/transport";
+import { RuntimeServiceProvider, TestRuntimeServices } from "../lib";
+import { RestClientMocker } from "../lib/RestClientMocker";
+
+const serviceProvider = new RuntimeServiceProvider();
+let runtimeServices: TestRuntimeServices;
+let clientMocker: RestClientMocker<PublicRelationshipTemplateReferenceClient>;
+
+beforeAll(async () => {
+    runtimeServices = (await serviceProvider.launch(1))[0];
+    const client = (runtimeServices.transport.publicRelationshipTemplateReferences as any).getPublicRelationshipTemplateReferencesUseCase
+        .publicRelationshipTemplateReferencesController.client as PublicRelationshipTemplateReferenceClient;
+    clientMocker = new RestClientMocker(client);
+}, 30000);
+
+afterAll(() => serviceProvider.stop());
+
+afterEach(() => {
+    clientMocker.restore();
+});
+
+describe("PublicRelationshipTemplateReferences", () => {
+    test("should read the PublicRelationshipTemplateReferences", async () => {
+        const mockResponse = [
+            {
+                title: "a",
+                description: "Description",
+                truncatedReference: "Reference"
+            },
+            {
+                title: "b",
+                description: "Description",
+                truncatedReference: "Reference"
+            },
+            {
+                title: "c",
+                description: "Description",
+                truncatedReference: "Reference"
+            }
+        ];
+
+        clientMocker.mockMethod("getPublicRelationshipTemplateReferences", () => {
+            return ClientResult.ok(mockResponse);
+        });
+
+        const publicRelationshipTemplateReferences = await runtimeServices.transport.publicRelationshipTemplateReferences.getPublicRelationshipTemplateReferences();
+
+        expect(publicRelationshipTemplateReferences.value).toStrictEqual(mockResponse);
```

Comment:
```suggestion
        const publicRelationshipTemplateReferences = await runtimeServices.transport.publicRelationshipTemplateReferences.getPublicRelationshipTemplateReferences();
        expect(publicRelationshipTemplateReferences.isSuccess).toBe(true);
        expect(publicRelationshipTemplateReferences.value).toStrictEqual(mockResponse);
```

Reply:
I like the one without the success more because when there is an error we only see that success should be true. But when directly checking value we will see the actual error that occurred

Reply:
I actually like the one without the success check more. As for the success check in an error case, we will only get the message that isSuccess should be true. Without that check, the error that occurred is displayed.

Reply:
Okay, I think often we do have the isSuccess check, but it's no big issue for me.